### PR TITLE
Add Watermeters devices

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
Baylan and b-meter devices

Baylan missing decoder and example files

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Review by Jaime-Trinidad — COMMENTED** [review]
Information provided by device makers

**Review by johanstokking — APPROVED** [review]
Looks good. Please update the branch and run `make fmt`

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,741 @@
 {
   "name": "lorawan-devices",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (johanstokking) [inline package-lock.json:4]: Revert this file and respect tooling dependencies (and their version) in `README`

**Review by johanstokking — COMMENTED** [review]
This should have been caught by the continuous integration already, right?

**Comment by Jaime-Trinidad** [comment]
files reverted, and fixed on PR #219

**Pull request closed without merging**